### ISSUE TITLE
UCSD Ped2

### ISSUE THREAD

**Issue by joos2010kj** [tianyu0207/RTFM#71]
So, is the training set just composed of 6 random abnormal videos and 4 random normal videos from UCSD Ped2? And the remaining 18 videos of abnormal and normal videos are used as test set?

I just keep having doubts about how the model can be trained on such a tiny data b/c each of those 10 "videos" has between 120 and 180 frames.

**Comment by coranholmes** [comment]
Have you got any idea? My results are quite bad on this dataset but can achieve similar results with the paper on other 3 datasets...
